singularity/Singularity.amd_aocl

```diff
@@ -83,65 +81,67 @@ From: fedora:41
-	dnf -y install patch sudo yum-utils hostname
-	dnf -y install git svn wget vim nano
-	dnf -y install automake autoconf cmake make
-	dnf -y install gcc gcc-c++ gcc-gfortran
-	dnf -y install arpack-devel python-devel python3-devel lapack-devel libpng-devel libxml2-devel readline-devel ncurses-devel f2py bzip2-devel libicu-devel python3-scipy python-setuptools gsl gsl-devel gdal gdal-devel libpqxx libpqxx-devel libaio libaio-devel
-	dnf -y install bison flex ncurses tar bzip2 which gettext nomacs
-	dnf -y install cmake3 nodejs
+    dnf -y install patch sudo yum-utils hostname
+    dnf -y install git svn wget vim nano
+    dnf -y install automake autoconf cmake make
+    dnf -y install gcc gcc-c++ gcc-gfortran
+    dnf -y install arpack-devel python-devel python3-devel lapack-devel libpng-devel libxml2-devel readline-devel ncurses-devel f2py bzip2-devel libicu-devel python3-scipy python-setuptools gsl gsl-devel gdal gdal-devel libpqxx libpqxx-devel libaio libaio-devel
+    dnf -y install bison flex ncurses tar bzip2 which gettext nomacs
+    dnf -y install cmake nodejs
     dnf -y install swarp rclone bc
     dnf -y install ripgrep
     wget https://download.oracle.com/otn_software/linux/instantclient/2360000/oracle-instantclient-basic-23.6.0.24.10-1.el9.x86_64.rpm
     dnf -y install oracle-instantclient-basic-23.6.0.24.10-1.el9.x86_64.rpm
+    rm oracle-instantclient-basic-*.rpm
     # Necessary for nomacs
     sudo strip --remove-section=.note.ABI-tag /usr/lib64/libQt5Core.so.5
+    export make=`which make`
 
     mkdir -p $INSTALLDIR/patches
     cd $INSTALLDIR && git clone --single-branch -b fedora-py3 https://github.com/tikk3r/flocs.git
     mv $INSTALLDIR/flocs/patches/* $INSTALLDIR/patches/
     
-	# Install OpenBLAS
-	mkdir -p /opt/OpenBLAS
-	cd /opt/OpenBLAS
-	git clone https://github.com/xianyi/OpenBLAS.git src
-	cd src && git checkout $OPENBLAS_VERSION
-	mkdir build && cd build
-	cmake3 -DCMAKE_INSTALL_PREFIX=/opt/OpenBLAS -DTARGET=$OPENBLAS_TARGET -DUSE_THREAD=1 -DNUM_THREADS=256 -DNUM_CORES=256  -DBUILD_SHARED_LIBS=ON -DNO_AVX512=$NOAVX512 ..
-	make -j$J
-	make install
-	rm -rf /opt/OpenBLAS/src
+    # Install OpenBLAS
+    mkdir -p /opt/OpenBLAS
+    cd /opt/OpenBLAS
+    git clone https://github.com/xianyi/OpenBLAS.git src
+    cd src && git checkout $OPENBLAS_VERSION
+    mkdir build && cd build
+    cmake -DCMAKE_INSTALL_PREFIX=/opt/OpenBLAS -DTARGET=$OPENBLAS_TARGET -DUSE_THREAD=1 -DNUM_THREADS=256 -DNUM_CORES=256  -DBUILD_SHARED_LIBS=ON -DNO_AVX512=$NOAVX512 ..
+    make -j$J
+    make install
+    rm -rf /opt/OpenBLAS/src
 
     dnf -y install openmpi openmpi-devel texinfo
 
     #
-	# AMD installs
+    # AMD installs
     #
     mkdir /opt/aocl
     cd /opt/aocl
-    wget https://download.amd.com/developer/eula/aocl/aocl-4-0/aocl-linux-aocc-4.0.tar.gz
-    tar xf aocl-linux-aocc-4.0.tar.gz
-    cd aocl-linux-aocc-4.0
+    wget https://download.amd.com/developer/eula/aocl/aocl-5-0/aocl-linux-aocc-5.0.0.tar.gz
```

Comment:
Let's not update this version in this MR.

Reply:
(It's ok to open a separate MR for updating AOCL  though, I'd just like to track the performance change from it in a more isolated fashion).